core/deployment/src/main/java/io/quarkus/deployment/util/ServiceUtil.java

```diff
@@ -19,6 +20,8 @@
 /**
  */
 public final class ServiceUtil {
+    private static final Charset DEFAULT_CHARSET = StandardCharsets.UTF_8;
```

Comment:
I'm not going to block the PR for this but this is not really an improvement. It's already a constant and now to figure out the charset you have to look out for the constant.